src/main/java/uk/gov/hmcts/darts/audio/helper/TransformedMediaHelper.java

```diff
@@ -54,14 +57,23 @@ public UUID saveToStorage(MediaRequestEntity mediaRequest, BinaryData binaryData
 
     @Transactional
     public TransformedMediaEntity createTransformedMediaEntity(MediaRequestEntity mediaRequest, String filename,
-                                                               OffsetDateTime startTime, OffsetDateTime endTime) {
+                                                               OffsetDateTime startTime, OffsetDateTime endTime,
+                                                               Long fileSize) {
+        AudioRequestOutputFormat audioRequestOutputFormat = AudioRequestOutputFormat.MP3;
```

Comment:
Wondering if this should just be parameterised and passed through because the top level service is already doing this - but then there is also a todo comment added by Chris under `uk.gov.hmcts.darts.audio.service.impl.MediaRequestServiceImpl#updateAudioRequestCompleted`.

I'll leave you to decide what best to do there, given my comment and what you've seen so far.

Reply:
possibly - will wait and see what outcome of other work is - leaving for now